dassh/material.py

```diff
@@ -78,17 +79,23 @@ class Material(LoggedClass):
     PROP_NAME = dict(zip(['density', 'heat_capacity', 'viscosity', 'thermal_conductivity'], LBH15_PROPERTIES)) 
     
     def __init__(self, name, temperature=298.15, from_file=None,
-                 coeff_dict=None, use_lbh15 = False, lbh15_correlations = None):
+                 coeff_dict=None, lbh15_correlations = None,
+                 corr_dict=None, use_correlation = False):
         LoggedClass.__init__(self, 0, f'dassh.Material.{name}')
         self.name = name
         self.temperature = temperature
-        # Read data into instance; use again to update properties later
+        # Read data into instance; use again to update properties later       
         if from_file:
             self.read_from_file(from_file)
         elif coeff_dict:
             self._define_from_coeff(coeff_dict)
-        elif self.name in Material.MATERIAL_LBH.keys() and use_lbh15:
-            self._define_from_lbh15(lbh15_correlations)
+        elif use_correlation and (self.name in ['sodium', 'nak'] or self.name in Material.MATERIAL_LBH.keys()):
```

Comment:
To avoid nested if statement, I'd suggest to make 2 "elif" statements, one checking on `use_correlation and self.name in ['sodium', 'nak']` and the other on `use_correlation and self.name in Material.MATERIAL_LBH.keys()`. Please, pay attention to the indentation.